vanilla-tests/src/main/java/fixtures/nonamedresponsetypes/models/HeadersResponseDurationHeaders.java

```diff
@@ -25,7 +25,8 @@ public final class HeadersResponseDurationHeaders {
      * @param rawHeaders The raw HttpHeaders that will be used to create the property values.
      */
     public HeadersResponseDurationHeaders(HttpHeaders rawHeaders) {
-        if (rawHeaders.getValue("value") != null) {
+        String value = rawHeaders.getValue("value");
+        if (value != null) {
```

Comment:
This may create unnecessary noise in client library code re-generation showing as diff. We should just keep the previous code as-is. Also, the local variable is not used in the line below, so, `rawHeaders.getValue()` is called still called twice.